patches/minecraft/net/minecraft/entity/item/BoatEntity.java.patch

```diff
@@ -9,7 +32,23 @@
                             ++k1;
                          }
                       }
-@@ -778,6 +778,20 @@
+@@ -688,13 +674,9 @@
+                if (!this.field_70170_p.field_72995_K && !this.field_70128_L) {
+                   this.func_70106_y();
+                   if (this.field_70170_p.func_82736_K().func_223586_b(GameRules.field_223604_g)) {
+-                     for(int i = 0; i < 3; ++i) {
+-                        this.func_199703_a(this.func_184453_r().func_195933_b());
++                     for(net.minecraft.item.ItemStack itemStack : this.func_184453_r().getDrops()) {
++                        this.func_199701_a_(itemStack);
+                      }
+-
+-                     for(int j = 0; j < 2; ++j) {
+-                        this.func_199703_a(Items.field_151055_y);
+-                     }
+                   }
```

Comment:
This isn't *bad*, but it seems like a area where Mojang hasn't moved to loot tables yet. How hard would it be to do that on our end instead of hardcoding.